vignettes/secuTrialR-package-vignette.Rmd

```diff
@@ -1,172 +1,570 @@
+still use the generic functions on it.
+Let's say you would only like to look at the recruitment plot for Bern alone.
+\vspace{5pt}
+
+```{r, fig.height = 3.8, fig.width = 8}
+# keep only Bern
+ctu05_data_bern <- subset_secuTrial(ctu05_data, centre = "Inselspital Bern (RPACK)")
+# plot
+plot_recruitment(ctu05_data_bern)
+```
+
+... or Bern and Berlin.
+\vspace{5pt}
+
+```{r, fig.height = 3.8, fig.width = 8}
+# keep only Bern and Berlin
+ctu05_data_bern_berlin <- subset_secuTrial(ctu05_data,
+                                           centre = c("Inselspital Bern (RPACK)",
+                                                      "Charité Berlin (RPACK)"))
+# plot
+plot_recruitment(ctu05_data_bern_berlin)
+```
 
+# The `as.data.frame` function
+
+This vignette has been working with the `secuTrialdata` object as a `list`.
+For some users, working with a `list` can be tiresome so `secuTrialR` provides an
+`as.data.frame()` method to save the `data.frame`s in the `secuTrialdata` object to
+an environment of your choice.
+
+Let's have a look at the state of your `globalenv()`
+before running `as.data.frame()`...
+\vspace{5pt}
+
+```{r}
+ls(globalenv())
 ```
 
-First we will asses the completeness only for "savedforms". Below you see
-that "age" is missing once while "grouping" and "bmd" are both missing twice
-just as reported in the validation overview.
+```{r}
+# add files to global environment
+as.data.frame(ctu05_data)
+```
 
+... and afterwards.
+\vspace{5pt}
 ```{r}
-secuTrialR:::assess_form_variable_completeness(form = sT_export_longtnames$dem00bmd, 
-                                  casenodes_table = sT_export_longtnames$casenodes,
-                                  validation_overview = val_ovv, 
-                                  completeness = "savedforms")
+ls(globalenv())
 ```
 
-If you would like to know the completeness for all forms also taking forms into account
-which have not been saved, then you need to set the completeness parameter to "allforms".
-Note that the occ_in_vp parameter has also been set to five. This is because the patients
-in the study were followed up, up to five times (i.e. five visits). Thus, if a patient 
-was followed up less then five times his or her data is not complete. To set this parameter 
-correctly, knowledge of the study design is necessary. As expected the completeness is 
-reduced when regarding all forms.
+# A note on `mnp*` variables
+
+There is a plethora of variables in the tables of secuTrial exports whose names start
+with `mnp`. These are meta variables which are e.g. important to logically link the
+different tables. Explaining them all is beyond the scope of this vignette.
+For detailed explanations please refer to the secuTrial user manual.
+
+# Disclaimer
+
+The descriptions of the export data and the logic within are our understanding of 
+them and come with no warranty. For in depth details please refer to the original
+secuTrial manuals.
+
+# The `sessionInfo()` of the vignette
```

Comment:
How about:

> sessionInfo() used to genarate this vignette

Reply:
I think its the `sessionInfo()` at the end of the vignette. Thus I don't quite agree. I'll change it to just "The `sessionInfo()`" though.